models/dex/aggregator/dex_aggregator_trades.sql

```diff
@@ -8,34 +8,60 @@
 }}
 SELECT *
 FROM
-    (
-        SELECT
-            blockchain,
-            project,
-            version,
-            block_date,
-            block_time,
-            token_bought_symbol,
-            token_sold_symbol,
-            token_pair,
-            token_bought_amount,
-            token_sold_amount,
-            token_bought_amount_raw,
-            token_sold_amount_raw,
-            amount_usd,
-            token_bought_address,
-            token_sold_address,
-            taker,
-            maker,
-            project_contract_address,
-            tx_hash,
-            tx_from,
-            tx_to,
-            trace_address,
-            evt_index
-        FROM {{ ref('cow_protocol_trades') }}
+(
+    SELECT  blockchain
+        ,project
+        ,version
+        ,block_date
+        ,block_time
+        ,token_bought_symbol
+        ,token_sold_symbol
+        ,token_pair
+        ,token_bought_amount
+        ,token_sold_amount
+        ,token_bought_amount_raw
+        ,token_sold_amount_raw
+        ,amount_usd
+        ,token_bought_address
+        ,token_sold_address
+        ,taker
+        ,maker
+        ,project_contract_address
+        ,tx_hash
+        ,tx_from
+        ,tx_to
+        ,trace_address
+        ,evt_index
+    FROM {{ ref('cow_protocol_trades') }}
+
+    UNION ALL
+
+    SELECT  blockchain
+        ,project
+        ,version
+        ,block_date
+        ,block_time
+        ,maker_symbol           AS token_bought_symbol
+        ,taker_symbol           AS token_sold_symbol
+        ,token_pair
+        ,maker_token_amount     AS token_bought_amount
+        ,taker_token_amount     AS token_sold_amount
+        ,maker_token_amount_raw AS token_bought_amount_raw
+        ,taker_token_amount_raw AS token_sold_amount_raw
+        ,amount_usd
+        ,maker_token            AS token_bought_address
+        ,taker_token            AS token_sold_address
```

Comment:
Hi @bakabhai993, do you think we can move the column renaming into the `zeroex_api_deduped_fills` model? So we can use the JINJA syntax to simplify the `dex_aggregator_trades` meta-table like `dex_trades` in the future.

Reply:
I am able to remove the renaming of the columns in this specific file, however, I would prefer to retain the current column names, specifically the **_zeroex_api_deduped_fills_** column. Would this be acceptable to you?

Reply:
I have already structured the zeroex_api_deduped_fills table on the lines of  dex_aggregator_trades FYI.

Reply:
Please have a look to [this file](https://github.com/duneanalytics/spellbook/blob/main/models/dex/dex_trades.sql), if we move the renaming (`select xxx as xxx`) in to the sub-model, we may have the code in the meta-table clearer.
Moreover if `zeroex_api_deduped_fills` is a model need to merge into `dex_aggregator.trades` meta-table, its schema can follow the meta-table's schema(covers all columns of the meta-table but not limited to). If we want to retain the current column names in the `zeroex_api_deduped_fill`, we can have columns having different names but the same value in the schema.

eg.
```
SELECT
COALESCE(b.maker_symbol,b.maker_token) AS maker_symbol,
COALESCE(b.maker_symbol,b.maker_token) AS token_bought_symbol
```
In this way `SELECT token_bought_symbol FROM dex_model` still works.

Reply:
Or we can have a wrapper model

Reply:
_**If we want to retain the current column names in the zeroex_api_deduped_fill, we can have columns having different names but the same value in the schema.**_

this should do the trick.

Reply:
Done.

Reply:
@RantumBits for your reference, we are adding extra columns in the deduped table for the reasons stated above.
@Hosuke thank you so much for the changes.